extensions/amp-app-banner/0.1/amp-app-banner.js

```diff
@@ -396,7 +396,7 @@ export class AmpAndroidAppBanner extends AbstractAppBanner {
         .then(response => this.parseManifest_(response))
         .catch(error => {
           this.hide_();
-          rethrowAsync(error);
+          rethrowAsync(dev().assertError(error));
```

Comment:
It's necessary for the linter. It's easy to see a call to `#assertError` and know that the return value is definitely an Error instance.